canonical/ubuntu.com
# WD-12059 - rebrand /cloud/private-cloud

## Done

Rebrand /cloud/multi-cloud. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14082.demos.haus/cloud/multi-cloud)
- [figma](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/24.10-u.c%2Fcloud?node-id=13-4246&t=LABk7mh5QO2QgsUd-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12059](https://warthogs.atlassian.net/browse/WD-12059)
